pyseir/__init__.py

```diff
@@ -1,3 +1,40 @@
 import pathlib
-DATA_DIR = pathlib.Path(__file__).parent.parent / 'data'
-OUTPUT_DIR = pathlib.Path(__file__).parent.parent / 'output'
+import os, math
+
+import multiprocessing.pool
+
+DATA_DIR = pathlib.Path(__file__).parent.parent / "data"
+OUTPUT_DIR = pathlib.Path(__file__).parent.parent / "output"
+
+
+def divide_up_pool():
```

Comment:
Should this code live here? Seems like `libs/multiprocess_utils.py` or something might be more appropriate